### PR TITLE
PLT-1121: One element, non-ordered lists and one element, ordered, ne…

### PR DESCRIPTION
**Pull request by florianorben**
…sted lists aren't rendered correctly

### CODE REVIEW

**Comment by it33** [comment]
Sweet! Thanks!!

**Comment by jwilander** [comment]
+1

**Comment by jwilander** [comment]
Closer please resolve https://mattermost.atlassian.net/browse/PLT-1121

**Comment by crspeller** [comment]
@hmhealey Please take a look at this.

**Comment by florianorben** [comment]
FYI, most of this is just pulling in the most recent changes from marked.js' code into list()

File: web/sass-files/sass/partials/_post.scss [hunk 1]
```diff
@@ -374,9 +374,9 @@ body.ios {
 	ul {
 		margin: 0;
 		padding: 0;
-		list-style: none;
```
Comment (hmhealey) [inline web/sass-files/sass/partials/_post.scss]: @crspeller Looks like you added this line when you were adding permalinks. Can you remember why?

**Comment by hmhealey** [comment]
Looks good other than the one thing for crspeller

File: web/sass-files/sass/partials/_post.scss (hunk 1 above)
Comment (crspeller) [inline web/sass-files/sass/partials/_post.scss]: @hmhealey When I merged the permalinks stuff I had merge conflicts with some of Assad's changes. This is probably just the result of a bad merge.

**Pull request merged**